src/map/ai/controllers/mob_controller.cpp

```diff
@@ -108,7 +108,7 @@ bool CMobController::CanPursueTarget(CBattleEntity* PTarget)
         if (!PMob->PAI->PathFind->InWater() && PTarget && !PTarget->StatusEffectContainer->HasStatusEffect(EFFECT_DEODORIZE))
         {
             // certain weather / deodorize will turn on time deaggro
-            return PMob->m_disableScent;
+            return !PMob->m_disableScent;
```

Comment:
hmm. `m_disableScent` should already be a bool that we're setting elsewhere. feels odd to return "not thisBool" here instead of setting it to be true/false as needed.

Rain has a few less documented "features" on retail as well, such as making worms more active in la theine and decreasing detection range for agro/linking separately from being able to track _after_  ..Gonna be a fun thing to try and measure on retail for somebody not named Teo or Tracent, right? 😄

Reply:
This isn't a request to do anything specific on my part. just an observation.